Code/Editor/EditorFramework/Assets/Implementation/AssetBrowserView.cpp

```diff
@@ -71,6 +74,122 @@ ezInt32 ezQtAssetBrowserView::GetIconScale() const
   return m_iIconSizePercentage;
 }
 
+void ezQtAssetBrowserView::dragEnterEvent(QDragEnterEvent* pEvent)
+{
+  if (pEvent->source())
+    pEvent->acceptProposedAction();
+}
+
+void ezQtAssetBrowserView::dragMoveEvent(QDragMoveEvent* pEvent)
+{
+  pEvent->acceptProposedAction();
+}
+
+void ezQtAssetBrowserView::dragLeaveEvent(QDragLeaveEvent* pEvent)
+{
+  pEvent->accept();
+}
+
+void ezQtAssetBrowserView::dropEvent(QDropEvent* pEvent)
```

Comment:
When is this codepath ever executed?

Also, why does this drop handler only copy files, and the other one on the widget triggers an import?

Reply:
So when my junior developer put this together we spent a while confused by this. From what we could find the drop events for internal drag and drop only triggers on the view but external drag and drop only triggers on the widget. 

Due to this is means this will only be used for moving files within the asset browser such as drag asset into a folder.

Reply:
Okay, i'll give that another try tomorrow.